src/internal/components/ThemeEditor.tsx

```diff
@@ -143,7 +146,7 @@ export const ThemeEditor = (props: ThemeEditorProps) => {
         ? [themeData, ...themeSaveState.history]
         : [...themeSaveState.history];
       const themeInitialHistory = {
-        history: history,
+        histories: history,
```

Comment:
All of line 143-164 needs to be updated to match what we do in loadPuckInitialHistory